ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/Levels/ApiGetLevelsByBadgeId.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models.DTOs;
+using ferrilata_devilline.Services.Interfaces;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using AutoMapper;
+using ferrilata_devilline.Models.DAOs;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios.ApiControllers.Levels
+{
+    [Collection("BaseCollection")]
+    public class ApiGetLevelsByBadgeId
+    {
+        private readonly TestContext _testContext;
+        private readonly ITokenService _tokenService;
```

Comment:
This filed is only used in constructor to assign value to token field and can be omitted.

Reply:
it is eaier to read this way